src/components/tagList/AposTagList.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="apos-tag-list">
+    <h3 class="apos-tag-list__title">{{ title }}</h3>
+    <ul class="apos-tag-list__items">
+      <AposTagListItem 
+        v-for="tag in tags" 
+        v-bind:key="tag.slug"
+        v-on:click="toggle"
+        :tag="tag"
+      />
+    </ul>
+  </div>
+</template>
+
+<script>
+import AposTagListItem from './AposTagListItem.vue';
+export default {
+  props: {
+    tags: Array,
+    title: {
+      default: 'Tag List',
+      type: String
+    },
+
+  },
+  components: { AposTagListItem },
+  data() {
+    return {
+      active: []
+    }
+  },
+  methods: {
+    toggle(slug) {
+      // could do something with `slug` here
```

Comment:
Maybe we should simply emit with the slug included. But not critical for now.

Reply:
emitted up to the parent